fbx/FBX.g4

```diff
@@ -0,0 +1,59 @@
+/*
+ * Grammar for text-based (AKA ASCII) encoding of FBX format,
+ * a format originally developed by Kaydara but now owned by Autodesk.
+ *
+ * This simpler parser just recognises the recursive node structure
+ * and does nothing to help you know which nodes can be inside which.
+ */
+grammar FBX;
+
+// PARSER RULES
+
+start
+    : node+
+    ;
+
+// Recursive node structure
+node
+    : name=IDENTIFIER
+      attributes
+      ( OPEN_BRACE
+        node*
+        CLOSE_BRACE
+      )?
+    ;
+
+attributes
+    : ( attribute
+        ( COMMA
+          attribute
+        )*
+      )?
```

Comment:
Please remove optionality and use `attributes?` in use-sites.

```suggestion
    : attribute (COMMA attribute)*
```

Reply:
Part of the definition of attributes is that there can be zero of them. I guess this is similar to the comment I had about the same suggestion on integerList and the others.